src/new-client/src/app/site/[site]/site-page.test.tsx

```diff
@@ -74,4 +74,19 @@ describe('Site Page', () => {
       screen.queryByRole('link', { name: 'Site Management' }),
     ).not.toBeInTheDocument();
   });
+
+  // TODO: Maybe parameterise these tests over permission/card pairs
```

Comment:
Would it be worth doing this now as I assume we will be getting more cards? If we aren't having any more cards will we ever come to do this and therefore do we need the comment?

Reply:
I did consider it, but the other two cards are currently permission-gated but this one isn't. 
I'm expecting the parameterised tests to involve the permissions (i.e. something like it.each("appears when user has permission"(card, permission) and another "it.each("doesn't appear when user doesn't")(card) etc. ) so I'd like to do the parameterisation once all 3 cards have permission gating.